### DISCUSSION TITLE
没有收到激活码！

### DISCUSSION THREAD

**Discussion by Jianliang-ouc** [getlantern/lantern-forum-cn#623] [General]

**Comment by MaxMKMax** [comment]
@Jianliang-ouc https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号又想到这个链接. 谢谢. 
您发完后请自行删掉个您的个人资料和单据.
